scripts/main.js

```diff
@@ -201,15 +199,14 @@ document.addEventListener('DOMContentLoaded', () => {
       sapien eu velit porttitor semper. Aliquam sed elementum enim.
       Suspendisse ultrices quis enim at gravida. Ut lectus urna, cursus
       et tellus in, faucibus lacinia urna. Morbi nibh purus, vehicula at`,
-      featuredImage: "./images/pictures/Snapshoot-Portfolio" + featureImageType,
-      featuredImageAlt: "Rock-Paper-Scissors Image",
+      featuredImage: `./images/pictures/Snapshoot-Portfolio${featureImageType}`,
+      featuredImageAlt: 'Rock-Paper-Scissors Image',
       technologies: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
-      liveVersion: "https://tadesse-alemayehu.github.io/YouTube-Clone/",
-      source: "https://github.com/Tadesse-Alemayehu/YouTube-Clone",
-    }
+      liveVersion: 'https://tadesse-alemayehu.github.io/YouTube-Clone/',
+      source: 'https://github.com/Tadesse-Alemayehu/YouTube-Clone',
+    },
   ];
 
   const ActionButtons = recentWorkSection.querySelectorAll('.action');
   ActionButtons.forEach((actionButton, index) => actionButton.addEventListener('click', () => showModal(Projects[index])));
-  console.log(ActionButtons.length);
 });
```

Comment:
- [ ] Good job so far aligning the buttons 👍 but the buttons break between the widths of 992px and 1241px. Kindly address this issue for a better user interface.
- Your page:
![Annotation 2022-03-02 084714](https://user-images.githubusercontent.com/85113567/156317910-0a0d1829-2c3f-44b6-adb3-9f598b2aa0e4.jpg)

Reply:
- It's not a good practice to implement the close button on the image. It would be better to add it separately on the top of your popup.

Reply:
@medaminedev66 I have added a close button for the mobile display https://github.com/Tadesse-Alemayehu/Portfolio/pull/14/commits/2e9d3d9224fcf6da0c0b2a2bc61ba5b8bc9c04f1